charts/nominatim/templates/updateJob.yaml

```diff
@@ -42,8 +42,11 @@ spec:
             {{- include "common.tplvalues.render" (dict "value" .Values.nominatim.extraEnv "context" $) | nindent 12 }}
           {{- end }}
           command:
-            - nominatim
-            - replication
+            - /bin/bash
+            - -ec
+            - |
+                nominatim replication --check-for-updates
+                nominatim replication --threads {{ .Values.nominatimReplications.threads }}
```

Comment:
This will output something like this:

```
2023-10-14 13:50:05: Using project directory: /nominatim
2023-10-14 13:50:29: New data available (3842 => 3850).
2023-10-14 13:50:30: Using project directory: /nominatim
2023-10-14 13:51:21  osm2pgsql version 1.9.2
...
...
```

I found it useful to see the state of pending updates first.